src/status_im/contexts/communities/events.cljs

```diff
@@ -169,6 +170,13 @@
                 {}
                 categories))}))
 
+(rf/reg-event-fx :communities/update-last-opened-at
+ (fn [_ [community-id]]
+   {:json-rpc/call [{:method     "wakuext_updateLastOpenedAt"
```

Comment:
Also, probably we should rename this end point. It somewhat gives idea that we are providing time at which community is opened.

Reply:
As Samuel mentioned in original issue, maybe something like `CommunityOpened`

Reply:
What do you think about `wakuext_communityUpdateLastOpenedAt`